### DISCUSSION TITLE
Prepopulate date(s)

### DISCUSSION THREAD

**Discussion by rependa** [primefaces/community#1227] [PrimeVue] [answered]
I'm looking for a way to prepopulate a date value into the calendar object, so that when the calendar popup is opened, it displays the selected date (or date range) defined in the input, Has anyone been able to do this? Is it possible?

**Comment by Uzeyir9695** [comment, marked as answer]
If you mean to set elected some dates by default, just pass the date array to Calendar v-model in format as it's passed in v-model.
without range, you can just pass single date format as it's passed to v-model without array.

**Reply by rependa** [reply]
Thank you. I think what was happening was my date object was being converted to a string.
